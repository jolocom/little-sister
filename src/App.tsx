import React from 'react'
import { Provider } from 'react-redux'
import { Navigator } from 'src/NavigatorContainer'
import { store } from 'src/store'
import { JolocomTheme } from 'src/styles/jolocom-theme'
import SplashScreen from 'react-native-splash-screen'

const { ThemeProvider } = require('react-native-material-ui')
const assign = require('object.assign/implementation')

Object.assign = assign

// tslint:disable-next-line: no-default-export
export default class App extends React.Component {
  componentDidMount() {
    SplashScreen.hide()
  }

  render() {
    return (
      <ThemeProvider uiTheme={JolocomTheme}>
        <Provider store={store}>
          <Navigator dispatch={store.dispatch} />
        </Provider>
      </ThemeProvider>
    )
  }
}
