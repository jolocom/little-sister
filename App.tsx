import * as React from 'react'
import {Provider} from 'react-redux'
import {View} from 'react-native'
import {RootStack} from './src/routes'
import {store} from './src/store'
import { ThemeProvider } from 'react-native-material-ui'
import { JolocomTheme } from 'src/styles/jolocom-theme'


// tslint:disable-next-line: no-default-export
export default class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <ThemeProvider uiTheme={JolocomTheme}>
          <RootStack />
        </ThemeProvider>
      </Provider>
    )
  }
}
