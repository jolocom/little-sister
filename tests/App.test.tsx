import * as React from 'react'
import App from 'src/App'

it('matches the snapshot', () => {
  const rendered = shallow(<App />)
  expect(rendered).toMatchSnapshot()
})
