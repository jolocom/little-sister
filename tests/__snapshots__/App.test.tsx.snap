// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<ThemeProvider
  uiTheme={
    Object {
      "button": Object {
        "container": Object {
          "backgroundColor": "#942f51",
          "borderRadius": 4,
          "margin": 0,
          "paddingBottom": 16,
          "paddingLeft": 32,
          "paddingRight": 32,
          "paddingTop": 12,
        },
      },
      "contentFontFamily": "Roboto, sans-serif",
      "palette": Object {
        "accent1Color": "#942f51",
        "accent2Color": "#942f51",
        "accent3Color": "#942f51",
        "accentColor": "#05050d",
        "backgroundCard": "#fafafa",
        "backgroundContainer": "#f5f5f5",
        "borderColor": "#e0e0e0",
        "canvasColor": "#ffffff",
        "disabledColor": "#9ba0aa",
        "primary1Color": "#b3c90f",
        "primary2Color": "#b3c90f",
        "primary3Color": "#b3c90f",
        "primaryColor": "#942f51",
        "primaryColorBlack": "#05050d",
        "primaryColorGrey": "#fafafa",
        "primaryColorSand": "#ffefdf",
        "primaryColorWhite": "white",
        "textColor": "#4b132b",
        "textColor_darkBrown75": "#633c38",
        "textColor_grey": "#a4a4a3",
        "textColor_silverGrey": "#9ba0aa",
      },
      "textStyles": Object {
        "contentInputFields": Object {
          "color": "#4b132b",
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "400",
        },
        "dark": Object {
          "appHeader": Object {
            "color": "white",
            "fontSize": 22,
          },
          "header": Object {
            "color": "white",
            "fontSize": 34,
          },
          "labelInputField": Object {
            "color": "#fafafa",
            "fontSize": 14,
          },
          "subheader": Object {
            "color": "#ffefdf",
            "fontSize": 30,
          },
          "tabHeader": Object {
            "active": Object {
              "color": "#ffefdf",
              "fontSize": 20,
            },
            "inactive": Object {
              "color": "#fafafa",
              "fontSize": 20,
            },
          },
          "textInputField": Object {
            "color": "white",
            "fontSize": 18,
          },
          "textL": Object {
            "color": "#ffefdf",
            "fontSize": 20,
          },
          "textM": Object {
            "color": "#ffefdf",
            "fontSize": 18,
          },
          "textS": Object {
            "color": "#ffefdf",
            "fontSize": 14,
          },
        },
        "headline": Object {
          "color": "#a4a4a3",
          "fontSize": 24,
          "fontStyle": "normal",
          "fontWeight": "200",
        },
        "inputFields": Object {
          "color": "#9ba0aa",
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "400",
        },
        "labelButton": Object {
          "color": "#942f51",
          "fontSize": "14px",
          "fontStyle": "normal",
          "fontWeight": "500",
          "textTransform": "uppercase",
        },
        "labelInputFields": Object {
          "color": "#633c38",
          "fontSize": 13,
          "fontStyle": "normal",
          "fontWeight": "300",
        },
        "light": Object {
          "labelDisplayField": Object {
            "color": "#942f51",
            "fontSize": 16,
          },
          "textDisplayField": Object {
            "color": "#05050d",
            "fontSize": 20,
          },
        },
        "screenHeader": Object {
          "color": "#4b132b",
          "fontSize": "20px",
          "fontStyle": "normal",
          "fontWeight": "500",
        },
        "sectionheader": Object {
          "color": "#4b132b",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "400",
        },
        "subheadline": Object {
          "color": "#a4a4a3",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "200",
        },
        "textCopy": Object {
          "color": "#a4a4a3",
          "fontSize": 13,
          "fontStyle": "normal",
          "fontWeight": "300",
        },
        "userName": Object {
          "color": "#4b132b",
          "fontSize": "28px",
          "fontStyle": "normal",
          "fontWeight": "400",
        },
      },
    }
  }
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(NavigatorContainer)
      dispatch={[Function]}
    />
  </Provider>
</ThemeProvider>
`;
